File: QViz.NET/Periscope.QViz/JSON/ToolSlack.cs
namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Tool Configuration Data for Slack Communicator
	/// </summary>
	public class ToolSlack
	{
		/// <summary>
		/// Name of the Tool Configuration (SLACK)
		/// </summary>
		public string toolName;

		/// <summary>
		/// URL of the Slack Hooks for sending Messages
		/// </summary>
		public string url;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public ToolSlack()
		{
			this.toolName = "";
			this.url = "";
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/ToolSlack.java
package com.periscope.qviz.json;

/**
 * JSON Model for Tool Configuration Data for Slack Communicator
 */
public class ToolSlack {

	/**
	 * Name of the Tool Configuration (SLACK)
	 */
	public String toolName;

	/**
	 * URL of the Slack Hooks for sending Messages
	 */
	public String url;

	/**
	 * Default Constructor
	 */
	public ToolSlack() {
		this.toolName = "";
		this.url = "";
	}

}
